README.md

```diff
@@ -111,6 +111,7 @@ would love to see as applications:
 
 ### Benefits
 
+- Up to $50,000 in funding per team/project
```

Comment:
Maybe saying per application could be more accurate ?

Reply:
agreed

Reply:
```suggestion
- Up to $50,000 in funding per application
```